dashboards/MySQL/MySQL_Replication_Summary.json

```diff
@@ -3,32 +3,32 @@
     "list": [
       {
         "builtIn": 1,
-        "datasource": "-- Grafana --",
+        "datasource": {
+          "type": "datasource",
+          "uid": "grafana"
+        },
```

Comment:
let's revert it

Reply:
the same in cases below

Reply:
https://www.notion.so/percona/How-to-create-a-new-PMM-dashboard-dddd892847974d5abed7bca79e7e7fa6?pvs=4#3f4edce9b6e64a37bf40c801dd8493f3

those uids changed multiple times already, we used to have prometheus, then Metrics and now it's grafana.

Reply:
I guess the one mentioned in the documentation has `"type": "prometheus"`, and this one has `"type": "grafana"`. I don't know if they are any different, but could be.

I'm not an expert, so why don't we ask @YashSartanpara1 to suggest the proper syntax?